### DISCUSSION TITLE
switch520 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#67] [模板讨论]
switch520 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by xlon72** [comment]
网站换了新地址不能用了，新地址：xxxxx520.com

**Reply by wjf0214** [reply]
> 网站换了新地址不能用了，新地址：xxxxx520.com

更新了模板，域名改为可填写模式。可以试试。
